source/type/pb_converter.cpp

```diff
@@ -49,6 +49,636 @@ namespace nt = navitia::type;
 namespace pt = boost::posix_time;
 namespace gd = boost::gregorian;
 
+//*********************************************
+namespace ProtoCreator{
+
+template<typename NT, typename PB>
+static void fill_codes(const NT* nt, const nt::Data& data, PB* pb) {
```

Comment:
maybe put it in the Filler will remove the data parametter. You can put this method private in Filler, and only forward declare it in the .h